tuist/tuist
# Ensure the latest semantic version is used when running via tuistenv

### Short description 📝

Running tuist via tuistenv can sometimes use an outdated version. 

_(e.g. I had version 1.9.0 installed locally, and it kept selecting that version over the latest version 1.12.0)_

### Solution 📦

Ensure versions are sorted ahead of selecting the last (latest) version.

### Implementation 👩‍💻👨‍💻

- [x] Ensure versions are sorted

### Test Plan 🛠

- Run `tuist install 1.9.0"
- Run `tuist install 1.12.0"
- Run `tuist version`
- Verify version 1.12.0 is used
